cg/meta/store/base.py

```diff
@@ -29,26 +35,110 @@ def add_new_analysis(bundle_data, case_obj, status, version_obj):
 
 def reset_case_action(case_obj):
     """ Resets action on case """
+
     case_obj.action = None
 
 
-def get_files(deliverables: dict, pipeline: str) -> dict:
+def get_files(deliverables: dict, pipeline: list) -> list:
     """Get all deliverable files from the pipeline"""
 
-    data = [
+    files = _get_files_non_index(deliverables, pipeline)
+    index_files = _get_files_index(deliverables, pipeline)
+
+    files.extend(index_files)
+
+    _check_mandatory_tags(files)
+    _convert_tags(files)
+
+    return files
+
+
+def _get_files_non_index(deliverables: dict, pipeline: list) -> list:
+    """ Get all files that are not index files from the deliverables file """
+
+    return [
         {"path": file["path"], "tags": get_tags(file, pipeline), "archive": False}
         for file in deliverables["files"]
     ]
 
-    return data
 
+def _get_files_index(deliverables: dict, pipeline: list) -> list:
+    """ Get all index files from the deliverables file """
 
-def get_tags(file: dict, pipeline: str) -> list:
+    return [
+        {"path": file["path_index"], "tags": get_tags(file, pipeline), "archive": False,}
```

Comment:
Should we add an index tag here? 
It looks like this in the current HK
`hugelymodelbat_lanes_1_sorted_md_brecal.cram.crai | hugelymodelbat, cram-index`

Reply:
I will update this.

Reply:
What index tags exist? Is it only `cram-index`?

Reply:
@henrikstranneheim  Can you check if I got the index tags correct? Is the list complete? I couldn't find anything for exphun.vcf.gz.csi

Suggested tags for index files:

| filename | tag |
| --- | --- |
| `...sorted_md_brecal.cram.crai` | `cram-index` |
| `...sorted_md_brecal_subsample_MT.bam.bai` | `bam-mt-index` |
| `..._exphun.vcf.gz.csi` | ??? |
| `..._comb.bcf.csi` | `sv-bcf-index` |
| `..._comb_ann_vep_parsed_ranked.vcf.gz.csi` | `vcf-sv-research-index` |
| `..._comb_ann_vep_parsed_ranked.selected.vcf.gz.csi` | `vcf-sv-clinical-index` |
| `..._gatkcomb.bcf.csi` | `gbcf-index` |
| `..._gatkcomb_rhocall_vt_af_frqf_cadd_vep_parsed_ranked.vcf.gz.tbi` | `vcf-snv-research-index` |
| `..._gatkcomb_rhocall_vt_af_frqf_cadd_vep_parsed_ranked.selected.vcf.gz.tbi` | `vcf-snv-clinical-index` |

Reply:
@henrikstranneheim does `vcf-str-index` make sense for expansionhunter files? The non index file tag is `vcf-str`. I can't find any examples in HK prod.

Reply:
yes it does or rather `str-vcf-index` and you will be fixing a bug when adding this